docs/setup.md

```diff
@@ -28,13 +28,25 @@ This document provides comprehensive instructions for setting up the project env
 
    <br>
 
-3. **Setting Up Environment Configuration**
+3. **Setting Up Environment Variables for the Testing User Credentials and URLs**
    <br>
 
-   - Duplicate [`template.env.json`](env_variables/template.env.json) file and rename the duplicated file to `local.env.json` in the env_variables folder
-   - Add your credentials and environment-specific variables to `local.env.json` from 1Password including api keys
-     **(Remember add all instances as in 1Password we have different vaults available for each namespace)**
-     <br>
+   Setting Up Environment Variables for Dev Clusters:
+
+   - Duplicate the file [devClusterTemplate.env.json](../env_variables/devClusterTemplate.env.json) and rename the duplicated file to `local.env.json` inside the `env_variables` folder.
+   - Include the required development namespace URLs for BRB/dBC/NBC.
+   - Test user data on development clusters are created using the school API.
+   - To retrieve the API keys for all three namespaces, navigate to 1Password (1PW).
+   - Contact QA team for the necessary 1Password links.
+
+   Setting Up Environment Variables for Staging:
```

Comment:
- this would be also be `h5` heading
- `Setting Up Environment Variables for Staging` -> `Setting Up Environment Variables for Staging Environment/Cluster`

Reply:
done!